nano/node/node.cpp

```diff
@@ -2669,7 +2669,12 @@ void nano::node::process_confirmed (std::shared_ptr<nano::block> block_a, uint8_
 	auto hash (block_a->hash ());
 	if (ledger.block_exists (block_a->type (), hash))
 	{
-		auto transaction (store.tx_begin_read ());
+		{
+			auto transaction (store.tx_begin_write ());
+			add_confirmation_heights (transaction, hash);
+		}
+
+		auto transaction (store.tx_begin_write ());
```

Comment:
I think this should be tx_begin_read  (like it was before)

Reply:
ahh good point, I must have changed that by accident again at some point, it's not necessary to write. Thanks